doc.html

```diff
@@ -47,104 +47,79 @@ <h2>4 produce_demand_forecaster Usage Scenarios</h2>
 
 The following sections describe some aspects of the use of produce_demand_forecaster. The examples were chosen to illustrate the basic operation of applications using produce_demand_forecaster, not to limit what produce_demand_forecaster may be used for.
 
-<h2>4.1 Irrigation</h2>
+<h2>4.1 Short supply chain producer</h2>
 
-The aim is to optimize water usage based on several data sources from produce_demand_forecaster. The irrigation decisions taken by a produce_demand_forecaster service could use a produce_demand_forecaster service to retrieve the historical data related to a specified region.
+The aim is to assist a short supply chain producer in making an informed cultivation
```

Comment:
Please don't make manual line breaks. The css will automatically take care of that (valid for the entire document)